static/app/components/nav/sidebar.tsx

```diff
@@ -0,0 +1,220 @@
+  border-right: 1px solid ${theme.translucentGray100};
+  background: #3e2648;
+  background: linear-gradient(180deg, #3e2648 0%, #442c4e 100%);
+  display: flex;
+  flex-direction: column;
+  z-index: ${theme.zIndex.sidebar};
+
+  @media screen and (min-width: ${p => p.theme.breakpoints.medium}) {
+    height: unset;
+    width: 74px;
+  }
+`;
+
+const SidebarItemList = styled('ul')`
+  position: relative;
+  list-style: none;
+  margin: 0;
+  padding: 0;
+  padding-top: ${space(1)};
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  color: rgba(255, 255, 255, 0.85);
+
+  @media screen and (min-width: ${p => p.theme.breakpoints.medium}) {
+    gap: ${space(1)};
+  }
+`;
+
+function SidebarItem({item}: {item: NavSidebarItem}) {
+  const location = useLocation();
+  const isActive = isNavItemActive(item, location);
+  const isSubmenuActive = isSubmenuItemActive(item, location);
+  const _to = resolveNavItemTo(item);
+  const to = _to ? makeLocationDescriptorFromTo(_to) : '#';
+
+  const FeatureGuard = item.feature ? Feature : Fragment;
+  const featureGuardProps: any = item.feature ?? {};
+
+  return (
+    <FeatureGuard {...featureGuardProps}>
+      <SidebarItemWrapper>
+        <Link
+          to={to}
+          className={isActive || isSubmenuActive ? 'active' : undefined}
+          aria-current={isActive ? 'page' : undefined}
+        >
+          {item.icon}
+          <span>{item.label}</span>
+        </Link>
+      </SidebarItemWrapper>
+    </FeatureGuard>
+  );
+}
+
+const SidebarItemWrapper = styled('li')`
+  svg {
+    --size: 14px;
+    width: var(--size);
+    height: var(--size);
+
+    @media (min-width: ${p => p.theme.breakpoints.medium}) {
+      --size: 18px;
+      padding-top: ${space(0.5)};
+    }
+  }
+  a {
+    display: flex;
+    flex-direction: row;
+    height: 32px;
+    gap: ${space(1.5)};
+    align-items: center;
+    padding: 0 ${space(1.5)};
+    color: var(--color, currentColor);
+    font-size: ${theme.fontSizeMedium};
+    font-weight: ${theme.fontWeightNormal};
+    line-height: 177.75%;
+    border: 1px solid transparent;
+
+    &:hover {
+      color: var(--color-hover, ${theme.white});
+    }
+
+    &.active {
+      color: var(--color-hover, ${theme.white});
+      background: rgba(255, 255, 255, 0.1);
+      border: 1px solid rgba(255, 255, 255, 0.06);
+    }
+
+    & > * {
+      pointer-events: none;
+    }
+
+    @media (min-width: ${p => p.theme.breakpoints.medium}) {
+      flex-direction: column;
+      justify-content: center;
+      height: 53px;
+      padding: ${space(0.5)} ${space(0.75)};
+      border-radius: ${theme.borderRadius};
+      font-size: ${theme.fontSizeExtraSmall};
+      margin-inline: ${space(1)};
+      gap: ${space(0.5)};
+    }
+  }
+`;
+
+const SidebarIndicator = styled(motion.span)`
+  position: absolute;
+  left: 0;
+  right: 0;
+  opacity: 0;
+  pointer-events: none;
+  margin-inline: ${space(1)};
+  width: 58px;
+  height: 53px;
```

Comment:
just want to confirm this is hardcoded to like the icon size? (otherise, anyway i could trick you into 54px? 🤣 - odd numbers always have a way of turning into fractions for me.)

Reply:
Yep, intentionally matching the icon size. Updated to `52px` in bb9d8210aa6 (since that is a nice, round `3.25rem`)